src/js/components/RangeSelector/EdgeControl.js

```diff
@@ -93,6 +119,29 @@ const EdgeControl = forwardRef(
           justify="center"
           alignSelf="stretch"
         >
+          {/* Add visually hidden range input for Safari VoiceOver */}
+          <VisuallyHiddenInput
+            type="range"
+            role="slider"
```

Comment:
Do we need both `type="range"` and `role="slider"`, from a quick google search it seems like one or the other should be used

Reply:
yes you are correct thank you removed `role="slider"` since we need the `type="range"` for Safari voice over!